.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: python
 python:
- - "2.7"
+ - 2.7
```

Comment:
What does listing the Python version here do? I've seen some projects list a Python version, while others leave it out entirely, instead relying on the `TOX_ENV` variable to construct the build matrix.